extensions/database-documentation/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "root": true,
```

Comment:
Remove this and use the same eslintrc we have in other extensions (except with different project path) so we get all the default eslint stuff for extensions automatically